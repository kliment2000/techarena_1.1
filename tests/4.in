3
10 3 8
4
1 a 1
1 b 2
2 b 2
3 c 2
3
1 a
1 b
3 c
1
1 2 a b
