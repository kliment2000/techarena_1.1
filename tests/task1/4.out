((1ab 2 {1.a 2.b}) 3c) 58.449999999999996
